core/app/models/spree/stock_movement.rb

```diff
@@ -5,13 +5,15 @@ class StockMovement < Spree::Base
 
     after_create :update_stock_item_quantity
 
-    validates :stock_item, presence: true
-    validates :quantity, presence: true, numericality: {
-                 greater_than_or_equal_to: -2**31,
-                 less_than_or_equal_to: 2**31-1,
-                 only_integer: true,
-                 allow_nil: true
-              }
+    with_options presence: true do
+      validates :stock_item
+      validates :quantity, numericality: {
+                  greater_than_or_equal_to: -2**31,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.